CIScripts/Test/Tests/README.md

```diff
@@ -0,0 +1,13 @@
+# How to run tests from local machine
+
+1. Deploy dev env using `deploy-dev-env` job.
```

Comment:
Is there a doc we can link to here?

Reply:
As far as I know, currently there is no such doc.